charts/argo-server/templates/service.yaml

```diff
@@ -1,10 +1,44 @@
+{{- $svcName := .Release.Name }}
+---
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ $svcName }}
 spec:
   selector:
     app.kubernetes.io/name: {{ .Release.Name }}
+  type: {{ if .Values.service.ingress.create }}NodePort{{ else }}ClusterIP{{ end }}
   ports:
     - port: {{ .Values.service.port }}
       targetPort: http
+      name: http
+{{- with .Values.service.ingress }}
```

Comment:
can this `with` block be moved up before the first `if` statement, since that uses the `.Values.service.ingress` piece as well? (So before line 10)

Reply:
Done